modules/ROOT/pages/questions/question-10.adoc

```diff
@@ -0,0 +1,18 @@
+10. the following plugins were removed from _TinyMCE 6.0_.
```

Comment:
@metricjs @EkimChau

Reply:
They were announced some time ago, left in place during the final few versions of 5 and are finally removed in 6
https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/